src/test/test_yaml.py

```diff
@@ -13,18 +13,20 @@
 import logging
 import unittest
 import os
-import ensembl.production.core.server_utils as su
+import pathlib
+
 from ensembl.production.core.config import load_config_yaml
 
 logging.basicConfig()
 
+here = pathlib.Path(__file__).parent.resolve()
 
 class YamlLoaderTest(unittest.TestCase):
     def test_parse_simple_str(self):
         home_dir = os.getenv("HOME")
         os.environ['EXPECTEDVAR'] = 'environmentVar'
         logging.info("Expected Home is %s" % home_dir)
-        config = load_config_yaml('./sample_config.yaml')
+        config = load_config_yaml(here/'sample_config.yaml')
```

Comment:
Potential typo here?

Reply:
Looks a bit too declarative for Python but that's actually how `pathlib` works. The `/` operator is overloaded for `Path` instances in order to work as concatenation: https://docs.python.org/3/library/pathlib.html#basic-use.